### PR TITLE
fix code by check

### PR DESCRIPTION
**Pull request by liu-to**

### CODE REVIEW

File: src/main/java/com/worksap/nlp/elasticsearch/plugins/chikkar/DictionaryManager.java [hunk 1]
```diff
@@ -342,7 +346,7 @@ public synchronized void addDictionary(RelationManager relationMgn, String dictP
                 }
             });
         } catch (IOException e) {
-            System.err.println(e);
+            log.error(e.getMessage());
```
Comment (kazuma-t) [inline src/main/java/com/worksap/nlp/elasticsearch/plugins/chikkar/DictionaryManager.java]: In this case, the user cannot know an error is happen. It is better to throw an exception, isn't it?

**Review by kazuma-t — CHANGES_REQUESTED** [review]

**Review by kazuma-t — APPROVED** [review]

File: src/main/java/com/worksap/nlp/elasticsearch/plugins/chikkar/DictionaryManager.java (hunk 1 above)
Comment (liu-to) [reply src/main/java/com/worksap/nlp/elasticsearch/plugins/chikkar/DictionaryManager.java]: Yes, you are right. Modified.

File: src/test/java/com/worksap/nlp/elasticsearch/plugins/chikkar/ChikkarTest.java [hunk 2]
```diff
@@ -108,27 +111,46 @@ public static void setUpBeforeClass() throws IOException {
 
         chikkar3 = new Chikkar(analyzer, false);
         chikkar3.loadDictionary(pathDirected.toString());
+
+        Path binPath = Paths.get(tempPath, "chikkarDict.bin");
+        FileOutputStream fileOut = new FileOutputStream(binPath.toFile());
```
Comment (kazuma-t) [inline src/test/java/com/worksap/nlp/elasticsearch/plugins/chikkar/ChikkarTest.java]: Use try-with-resources statements

**Review by kazuma-t — CHANGES_REQUESTED** [review]

File: src/test/java/com/worksap/nlp/elasticsearch/plugins/chikkar/ChikkarTest.java (hunk 2 above)
Comment (liu-to) [reply src/test/java/com/worksap/nlp/elasticsearch/plugins/chikkar/ChikkarTest.java]: OK, modified.

**Review by kazuma-t — APPROVED** [review]

**Pull request merged**